packages/design-system/index.css

```
@@ -0,0 +1,41 @@
+
+/* 1. Tokens — barrel import */
+@import './tokens/index.css';
+
+/* 2. Theme — Semantic mappings */
+@import './theme.css';
+
+/* 3. Utilities — Custom utilities + component styles */
+@import './utilities.css';
+
+/* ── Source paths for Tailwind scanning ──────────────────── */
+@source '../ui/**/*.{js,ts,jsx,tsx}';
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)